core/src/test/java/org/elasticsearch/action/search/MultiSearchRequestTests.java

```diff
@@ -167,6 +164,13 @@ public void testResponseErrorToXContent() throws IOException {
                 builder.string());
     }
 
+    public void testMaxParallelSearchRequests() {
```

Comment:
One more nit, if we could rename this to `testMaxConcurrentSearchRequests`; I promise no additional nits on this PR. :smile: